### PR TITLE
Fixed javadoc tags

### PR DESCRIPTION
**Pull request by tbw777**
A part of javadoc sync global superproject

I can fix every nb byte

### CODE REVIEW

**Comment by ebarboni** [comment]
hi @tbw777 would be nice if you can link the other PR about javadoc changes you made. 
@mbien this save me some times :p on the split of #4450.

**Comment by tbw777** [comment]
This PR changes type is independent of other

**Comment by ebarboni** [comment]
sorry was thinking on other PR you made about javadoc fixes like #5222.

File: ide/editor.lib/src/org/netbeans/editor/JumpList.java [hunk 1]
```diff
@@ -210,7 +210,7 @@ public static boolean hasNext() {
 
     /**
      * @return Unspecified string.
-     * @deprecate Should have never been public.
+     * @deprecated Should have never been public.
      */
```
Comment (mbien) [inline ide/editor.lib/src/org/netbeans/editor/JumpList.java:214]: please add the `@Deprecated` annotation too

File: ide/csl.api/src/org/netbeans/modules/csl/spi/DefaultLanguageConfig.java [hunk 2]
```diff
@@ -195,7 +195,7 @@ public EmbeddingIndexerFactory getIndexerFactory() {
      * 
      * @return true iff this language configuration provides a
      *  structure scanner.
-     * @wasdeprecated This function is not called anymore. You
+     * @deprecated This function is not called anymore. You
```
Comment (mbien) [inline ide/csl.api/src/org/netbeans/modules/csl/spi/DefaultLanguageConfig.java:198]: please add the `@Deprecated` annotation too

File: ide/csl.api/src/org/netbeans/modules/csl/spi/DefaultLanguageConfig.java [hunk 3]
```diff
@@ -111,7 +111,7 @@ public Parser getParser() {
      * NOTE: Code folding doesn't work until you enable code folding for your
      * editor kit; see GsfEditorKitFactory's reference to CODE_FOLDING_ENABLE for
      * an example.
-     * @wasdeprecated This function is not called anymore. You
+     * @deprecated This function is not called anymore. You
```
Comment (mbien) [inline ide/csl.api/src/org/netbeans/modules/csl/spi/DefaultLanguageConfig.java:114]: please add the `@Deprecated` annotation too

**Review by pepness — APPROVED** [review]
Looks good

**Review by mbien — APPROVED** [review]
thanks for the update. Looks good.

**Pull request merged**